AsyncDisplayKit/ASTextNode.mm

```diff
@@ -397,11 +401,12 @@ - (CGSize)calculateSizeThatFits:(CGSize)constrainedSize
     }
   }
   
-  //add textContainerInset
+  // Add the constrained size back textContainerInset
```

Comment:
This is actually adding back in the textContainerInset to the returned text size.